ui/components/app/confirm/info/row/alert-row.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { createContext } from 'react';
+import { Severity } from '../../../../../helpers/constants/design-system';
+import InlineAlert from '../../../confirmations/alerts/inline-alert/inline-alert';
+import useAlerts from '../../../../../hooks/useAlerts';
+import { ConfirmInfoRow, ConfirmInfoRowVariant } from './row';
+
+export type AlertRowProps = {
+  alertKey: string;
+  alertOwnerId: string;
+  children?: React.ReactNode;
+  label: string;
```

Comment:
Should we use a union here with the `ConfirmInfoRowProps` so we don't have to keep the two in sync and duplicate all the core row properties?

Reply:
Done.